### PR TITLE
Update Mimecast README.md

### PR DESCRIPTION
**Pull request by cherryleaf**
markdown syntax tidy up

Type of change

- Enhancement

markdown syntax tidy up

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/mimecast/changelog.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 - version: "0.0.9"
   changes:
-    - description: Update ecs to version 8.2.0 and implement better practice for email ECS fields.
+    - description: markdown syntax tidy up in readme
       type: enhancement
-      link: https://github.com/elastic/integrations/pull/2841
+      link: https://github.com/elastic/integrations/pull/2990
```
Comment (bmorelli25) [inline packages/mimecast/changelog.yml:5]: I don't think you have the most up to date version of these files locally. This PR needs to be merged with main and updated to version 0.0.10.

Comment (GinnyGalicia) [reply packages/mimecast/changelog.yml]: changelog updated

File: packages/1password/_dev/build/docs/README.md [hunk 2]
```diff
@@ -34,4 +34,4 @@ Uses the 1Password Events API to retrieve information about items in shared vaul
 
 {{fields "item_usages"}}
 
-{{event "item_usages"}}
+{{event "item_usages"}}
```
Comment (andrewkroh) [inline packages/1password/_dev/build/docs/README.md:37]: This should be reverted given this is a mimecast PR.

File: packages/mimecast/changelog.yml [hunk 3]
```diff
@@ -1,3 +1,8 @@
+- version: "0.0.10"
+  changes:
+    - description: markdown syntax tidy up in readme
```
Comment (andrewkroh) [inline packages/mimecast/changelog.yml:3]: These are rendered in our docs so I think it would be best to use a complete sentence.

**Review by andrewkroh — CHANGES_REQUESTED** [review]

File: packages/1password/_dev/build/docs/README.md (hunk 2 above)
Comment (ellis-elastic) [reply packages/1password/_dev/build/docs/README.md]: Done

File: packages/mimecast/changelog.yml (hunk 3 above)
Comment (GinnyGalicia) [reply packages/mimecast/changelog.yml]: I've edited the description

**Review by JezelMishap — APPROVED** [review]

**Review by andrewkroh — APPROVED** [review]

**Comment by andrewkroh** [comment]
It looks like @cherryleaf has not given permissions to allow pushing updates to this PR. I'll open a new PR with the same content + a merge.

**Pull request closed without merging**